app/models.py

```diff
@@ -471,3 +486,8 @@ def filter_null_value_fields(obj):
     return dict(
         filter(lambda x: x[1] is not None, obj.items())
     )
+
+
+def generate_new_service_id():
+    # TODO: Decide what we want G7 service IDs to look like and implement
```

Comment:
Is this comment still valid or have we chosen to use the `uuid4`? The method is being used by the model.